src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -97,7 +97,7 @@ public IEnumerable<PackageResult> list_run(ChocolateyConfiguration config)
             if (config.RegularOutput) this.Log().Debug(() => "Running list with the following filter = '{0}'".format_with(config.Input));
             if (config.RegularOutput) this.Log().Debug(() => "--- Start of List ---");
             foreach (var pkg in NugetList.GetPackages(config, _nugetLogger))
-
+

```

Comment:
actually a rebase is what we always hope for. I don't like merge commits anywhere except for explicit merges.